content/plugins/aws.md

```diff
@@ -10,7 +10,10 @@ weight: 100
 
 # Description
```

Comment:
I suggest you remove the description headline. It's pretty obvious that the first two lines are the description of the plugin.